### DISCUSSION TITLE
An error appears on my computer

### DISCUSSION THREAD

**Discussion by ghost** [nesbox/TIC-80#1550] [Q&A]
Hello, on my windows there's a problem when I open the executable it says "This application can't be run in your software, contact the developer or something", how do I fix it?

**Comment by nesbox** [comment]
The main version is 64 bit by default, please use `tic80-v0.90-winxp.zip`, it's 32 bit
https://github.com/nesbox/TIC-80/releases/download/v0.90.1723/tic80-v0.90-winxp.zip
